ui/node/attributes.go

```diff
@@ -217,12 +218,63 @@ type ScriptAttributes struct {
 	NodeType string `json:"node_type"`
 }
 
+// SelectAttributes represents the attributes of a select node
+//
+// swagger:model uiNodeSelectAttributes
+type SelectAttributes struct {
+	// The input's element name.
+	//
+	// required: true
+	Name string `json:"name"`
+
+	// The input's value.
+	FieldValue interface{} `json:"value,omitempty" faker:"string"`
```

Comment:
Do we need this field?

Reply:
Yes, that's the currently selected value. See remarks above.